microprofile.jdt/org.eclipse.lsp4mp.jdt.core/src/main/java/org/eclipse/lsp4mp/jdt/core/java/definition/AbstractAnnotationDefinitionParticipant.java

```diff
@@ -118,13 +120,11 @@ public boolean isAdaptedForDefinition(JavaDefinitionContext context, IProgressMo
 
 		// Try to get the annotation member value
 		String annotationSource = ((ISourceReference) annotation).getSource();
-		String annotationMemberValue = null;
-		for (String annotationMemberName : annotationMemberNames) {
-			annotationMemberValue = getAnnotationMemberValue(annotation, annotationMemberName);
-			if (annotationMemberValue != null) {
-				break;
-			}
-		}
+		AnnotationMemberInfo annotationMemberInfo = getAnnotationMemberAt(annotation, annotationSource,

```

Comment:
Remove annotationSource by moving in getAnnotationMemberAt

Theain idea is to have few parameters for the method.